subprojects/docs/src/docs/release/notes.md

```diff
@@ -343,6 +349,28 @@ The following Attributes have disambiguation rules defined.
 For more information, see [Attribute Disambiguation Rules](userguide/variant_attributes.html#sec:abm_disambiguation_rules).
 
 
+
+<a name="configuration"></a>
+### Configuration Cache
+
+#### Improved configuration cache failure recovery
```

Comment:
Piotr: It's not entirely clear to me after reading this what's the point of having config cache enabled / disabled. It seems the main point is automatic recovery from failures anyway.

Reply:
@mlopatkin can you address this?

Reply:
I see an explanation at the beginning of the _Configuration Cache_ section now and I also rephrased the recovery entry for clarity.